controller/machine/controller.go

```diff
@@ -41,6 +41,7 @@ func Register(management *config.ManagementContext) {
 		machineTemplateGenericClient: management.Management.MachineTemplates("").ObjectClient().UnstructuredClient(),
 		configMapGetter:              management.K8sClient.CoreV1(),
 		logger:                       management.EventLogger,
+		clusters:                     management.Management.Clusters(""),
```

Comment:
can we use cache in favor of making a direct api call? When use cache, use `clusterLister` as the name of the param